blocks/trending-news/trending-news.js

```diff
@@ -0,0 +1,113 @@
+import { getTrendingNews } from '../../scripts/mockapi.js';
+
+export default function decorate(block) {
+  const newsData = getTrendingNews();
+  block.textContent = '';
+
+  function createNewsItem(news) {
+    const article = document.createElement('div');
+    article.className = 'article';
+
+    const mediaWrapper = document.createElement('div');
+    mediaWrapper.className = 'picture-wrapper';
+
+    const picture = document.createElement('picture');
```

Comment:
We should use `createOptimizedPicture`

Reply:
this is the image coming from API , not published on edge delivery so `createOptimizedPicture` was not working or I am missing something

Reply:
used recently added `createOptamizedImage` from block utils.